<ui version="4.0" >
 <class>MultiLoadRDBBase</class>
 <widget class="QDialog" name="MultiLoadRDBBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Multi-module analysis - selection</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>RootDB file</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileName" />
       </item>
       <item>
        <widget class="QPushButton" name="BrowseButton" >
         <property name="text" >
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <widget class="QWidget" name="TabPage" >
        <attribute name="title" >
         <string>Analysis</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QCheckBox" name="runPDB" >
             <property name="text" >
              <string>run analysis</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <spacer name="spacer7" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="textLabel1_3" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="text" >
                <string>save output to</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="plotsPath" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="plotsBrwsButton" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="text" >
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QLabel" name="textLabel1_2" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <property name="text" >
                <string>cuts to apply:</string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cutsBox" >
               <property name="enabled" >
                <bool>false</bool>
               </property>
               <item>
                <property name="text" >
                 <string>use current cut set</string>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>FLEX cuts</string>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>LOAD (stave) cuts</string>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>LOAD (sector) cuts</string>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>STAVE cuts</string>
                </property>
               </item>
               <item>
                <property name="text" >
                 <string>DISK cuts</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="removeAfterPDB" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string>Remove data after PDB analysis</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="spacer12" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>41</width>
               <height>348</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab1" >
        <attribute name="title" >
         <string>Scans</string>
        </attribute>
        <widget class="QLabel" name="textLabel2" >
         <property name="geometry" >
          <rect>
           <x>12</x>
           <y>12</y>
           <width>462</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text" >
          <string>Select the associated type of scan for each data set:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab2" >
        <attribute name="title" >
         <string>Modules</string>
        </attribute>
        <widget class="QLabel" name="textLabel3" >
         <property name="geometry" >
          <rect>
           <x>12</x>
           <y>12</y>
           <width>462</width>
           <height>18</height>
          </rect>
         </property>
         <property name="text" >
          <string>Select module to be analysed</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QWidget" name="layout5" >
         <property name="geometry" >
          <rect>
           <x>12</x>
           <y>487</y>
           <width>462</width>
           <height>36</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QPushButton" name="allButton" >
            <property name="text" >
             <string>select all</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="spacer2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>81</width>
              <height>21</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="noneButton" >
            <property name="text" >
             <string>unselect all</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="OKButton" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer1" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>281</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>removeAfterPDB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotsPath</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>textLabel1_3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>runPDB</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotsBrwsButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
